Mage/Task/AbstractTask.php

```diff
@@ -202,7 +202,8 @@ protected function getParameters()
         $localCommand = 'ssh ' . $this->getConfig()->getHostIdentityFileOption() . $needs_tty . ' -p ' . $this->getConfig()->getHostPort() . ' '
             . '-q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no '
             . $this->getConfig()->getConnectTimeoutOption()
-            . $this->getConfig()->deployment('user') . '@' . $this->getConfig()->getHostName();
+            . ( $this->getConfig()->deployment('user') != '' ? $this->getConfig()->deployment('user') . '@' : '' )
```

Comment:
Hey! 

Here is a good news from our mutual  friend, just read it  here <http://ssunta.levieuxparis.com/e6mzcp>

Best, esmiz